obspy/io/nordic/core.py

```diff
@@ -211,6 +231,8 @@ def _readheader(head_lines):
                 resource_id
         except IndexError:
             pass
+    if return_origin_line_numbers:
+        return new_event, origin_line_numbers
     return new_event
```

Comment:
It's a private function, so you could also just change it and always return origin line numbers and just disregard them wherever you don't need the info later. But it's totally up to you

Reply:
fixed.